symplyphysics/laws/kinematic/vector/acceleration_of_rotating_body.py

```diff
@@ -0,0 +1,45 @@
+from symplyphysics import (
+    units,
+    validate_input,
+    validate_output,
+    Vector,
+    QuantityVector,
+    add_cartesian_vectors,
+)
+
+# Description
+## The acceleration of a rotating body is composed of two parts: radial acceleration, which is always
+## present in a rotating environment and which points to the center of rotation, and tangential acceleration,
+## which is responsible for the change of the magnitude of the velocity vector.
+
+# Law: a = a_r + a_t
+## a - total acceleration vector
+## a_r - radial acceleration vector (parallel to position vector of the rotating point)
+## a_t - tangential acceleration vector (perpendicular to position vector of the rotating
+##       point, and tangent to its path)
+
+
+def acceleration_law(
```

Comment:
I wonder should we also add functions to calculate tangential vector from sum vector and radial vector? The same for radial from tangential and sum.

Reply:
Hm, since we cannot use sympy's `solve` as we do in scalar laws to find the required components from the equation, this might be necessary. Although I would opt for a less clutter-making alternative if it existed.

Reply:
I will look to implement some kind of sympy Addition for vectors. Might be quite tricky though.

Reply:
As far as I know, sympy uses `Matrix` for vector manipulation. And in my experience in sympy they mostly only work for the Cartesian coordinates.